webpack.config.js

```diff
@@ -0,0 +1,35 @@
+var path = require('path');
+var SRC_DIR = path.join(__dirname, '/client/src');
+var DIST_DIR = path.join(__dirname, '/client/dist');
+const MomentLocalesPlugin = require('moment-locales-webpack-plugin');
+
+module.exports = {
+  entry: `${SRC_DIR}/index.jsx`,
+  output: {
+    filename: 'bundle.js',
+    path: DIST_DIR
+  },
+  module: {
+    loaders: [
+      {
+        test: /\.jsx?/,
+        include: SRC_DIR,
+        loader: 'babel-loader',
+        query: {
+          presets: ['react', 'es2015']
+        }
+      }
+    ]
+  },
+  plugins: [
```

Comment:
I'm not familiar with this plugin - I assume it's working as expected?

Reply:
I am planning to use it later as my module has a date element and I think it might be useful for displaying it in the right format. I also used moment.js when inserting the date from my seeding script into the database in the expected format